feat(i18n): Add Catalan translation

hugoovf opened the pull request:
Complete Catalan translation of the Kids Chores Dashboard interface, adapting all labels, messages, and interface elements to Catalan. The translation maintains child-friendly language while preserving the dashboard's functionality. All translatable text sections have been updated, including chore status labels, achievement descriptions, reward system terminology, admin controls, and parent dashboard elements.

hugoovf committed:
feat(i18n): Add Catalan translation

Complete Catalan translation of the Kids Chores Dashboard interface, adapting all labels, messages, and interface elements to Catalan. The translation maintains child-friendly language while preserving the dashboard's functionality. All translatable text sections have been updated, including chore status labels, achievement descriptions, reward system terminology, admin controls, and parent dashboard elements.

Rabosa616 commented:
Hello @hugoovf take a look, there are some spanish words like:

- Para -> Per
- Pendiente -> Pendent
- Diaria -> Diària
- Estado -> Estat
- Racha ->Ratxa
- Premios -> Premis
- Bonificaciones aplicadas -> Bonificacions aplicades
- Bonificación total -> Bonificació total
- Penalizaciones aplicadas -> Penalitzacions aplicades
- Penalización total -> Penalització total
- Logros -> Èxits
- Retos -> Reptes
- Aprobaciones -> Aprovacions
- Reclamaciones -> Reclamacions
- Acciones de administración de tareas -> Accions d'administració de Tasques
- Permitir múltiples reclamaciones -> Permetre múltiples reclamacions
- Fecha límite -> Data límit
- Ninguna -> Cap
- Repetición -> Repetició
- Intervalo -> Interval
- No hay tarea seleccionada o estado no disponible -> No hi ha cap tasca seleccionada o estat no disponible
- +Próxima fecha -> +Pròxima data
- +1 Día -> +1 Dia
- +1 Semana -> +1 Setmana
- Borrar fecha -> Esborrar data
- Seleccionar nueva fecha y hora -> Seleccionar nova data i hora
- (Toca para cambiar / Mantén para establecer) -> (Toca per canviar / Mantingues per establir)
- Restablecer todo -> Restablir-ho tot
- Tareas atrasadas -> Tasques endarrerides
- Restablecer estado de atraso -> Restablir estat d'endarreriment
- Días aplicables -> Dies aplicables
- Atrasada -> Endarrerida
- Ninguna -> Cap
- Sin fecha -> Sense data
- Criterios -> Criteris
- Fecha de inicio -> Data d'inici
- Fecha de fin -> Data de fi
- Progreso -> Progrés
- Asignado a -> Assignat a
- Tarea relacionada -> Tasca relacionada
- Meta de racha -> Meta de ratxa
- Tareas totales -> Tasques totals
- Mínimo diario -> Mínim diària
- Meta diaria -> Meta diària

hugoovf committed:
Fix translations in kc_dashboard_ca.yaml

Updated various labels and terms in the YAML file to reflect correct Catalan translations.

hugoovf commented:
Fixed, thanks @Rabosa616!

ccpk1 commented:
Thanks for the effort on this!

I’m going to close this PR because these translations have now been implemented via our new translation system in the latest release: [v0.5.0b3](https://github.com/ad-ha/kidschores-ha/releases/tag/0.5.0b3).

Apologies for the timing here—since we were in the middle of overhauling the localization system, we moved directly to the new approach rather than merging the old format.

Want to help manage this? If you spot any issues with the Catalan translations in the beta, please feel free to open an issue.

Better yet, if you are interested, I would love to add you as a Proofreader/Approver in our new Crowdin project. This would give you direct access to review, fix, and approve translations for the language. Let me know if you're interested and I'll get you set up!

hugoovf commented:
@ccpk1, hey np, and yeah I'm interested being a Crowdin proofreader for Catalan & Spanish (C1 Catalan, and native Spanish), I've the same username there.

ccpk1 commented:
I think you are setup now.  I'm not familiar with exactly how it works, but
let me know what you figure out and if you have trouble.

On Fri, Feb 6, 2026 at 4:14 PM hugoo ༄ ˚｡⋆★ ***@***.***>
wrote:

> *hugoovf* left a comment (ccpk1/kidschores-ha-dashboard#59)
> <https://github.com/ccpk1/kidschores-ha-dashboard/pull/59#issuecomment-3862593191>
>
> @ccpk1 <https://github.com/ccpk1>, hey np, and yeah I'm interested being
> a Crowdin proofreader, I've the same username there.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ccpk1/kidschores-ha-dashboard/pull/59#issuecomment-3862593191>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APNRZYFAFF53MAAQSZQ6U2D4KT72TAVCNFSM6AAAAACD4N7XAOVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTQNRSGU4TGMJZGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
